### PR TITLE
Include license file in the crate

### PR DESCRIPTION
**Pull request by kiilerix**
The actual license is not only relevant for contributing to the project, but even more for using the library. The license files should thus be shipped in the crate like most other crates do.

I would like to include whoami in Fedora as new dependency of Mercurial. That needs the license files in the crate.

(Also, most other rust projects seem to leave out the .txt extension and the license version from the file names. For consistency, it would thus be even better if the license files were named `LICENSE_APACHE`, `LICENSE_BOOST`, and `LICENSE_MIT`.)

### CODE REVIEW

**Comment by kiilerix** [comment]
I don't see anything in the failed checks that seems related to the changes.

File: Cargo.toml
Comment (AldaronLau) [inline Cargo.toml]: This line has gotten too long, can you make change the formatting, and also keep in alphabetical order?

```suggestion
include = [
    "LICENSE_APACHE_2_0.txt",
    "LICENSE_BOOST_1_0.txt",
    "LICENSE_MIT.txt",
    "README.md", 
    "src/*",
]
```

**Review by AldaronLau — CHANGES_REQUESTED** [review]
Thanks for the PR!  Feel free to rename the license files as well.

**Comment by AldaronLau** [comment]
> I don't see anything in the failed checks that seems related to the changes.

Agreed.  I'll try and see what's going on with it.

File: Cargo.toml
Comment (kiilerix) [reply Cargo.toml]: I will sort the list, but after renaming the files the line is shorter and there is not the same need to make it multi line.

(There could perhaps still be benefits in applying the full reformatting that happens anyway when packaging.)

**Review by AldaronLau — APPROVED** [review]

**Comment by kiilerix** [comment]
Thanks for merging. (But FWIW, I much prefer rebasing of my clean history instead of squashing.)

It would be convenient for me if the changes were available in an official 1.4.1 release. Any thoughts about when that will happen? Will you consider making a release just for this reason?

**Comment by AldaronLau** [comment]
> Thanks for merging. (But FWIW, I much prefer rebasing of my clean history instead of squashing.)

I exclusively use squash merging because it makes it easier for me as a maintainer.  Some contributors to some of my crates will have an unclean history containing useless commit messages such as "oops", "try again (again (again))", and "fix bug".  Everyone works differently (you used a force-push to keep commits clean, while I usually choose not to, so that history is preserved during the review process up until merge), and I don't want to force volunteers into cleaning up their history into something readable, or working a specific way.  So, I give everyone the same treatment, because it makes it easier for me when volunteering to maintain this project and other crates.  If this is something that would turn you away from making future contributions, I will re-think my stance.

> It would be convenient for me if the changes were available in an official 1.4.1 release. Any thoughts about when that will happen? Will you consider making a release just for this reason?

This is already in progress; The release will happen tonight Americas/Chicago time.

**Comment by AldaronLau** [comment]
Version 1.4.1 is now released with the patch.

**Pull request merged**
